www/js/search.js

```diff
@@ -663,6 +663,24 @@ module.exports = {
       });
   },
 
+  loadBani(BaniID) {
+    const $shabadList = this.$shabad || document.getElementById('shabad');
+    $shabadList.innerHTML = '';
+    banidb.loadBani(BaniID)
+      .then(rowsDb => {
+        const rows = rowsDb.map((rowDb) => {
+          let row = rowDb;
+          // when object from db is not a verse itself
+          if (rowDb.Verse) row = rowDb.Verse;
```

Comment:
nit picking: can we be consistent with if statements and do one liners with curly brackets as well?